soap/soap.go

```diff
@@ -152,6 +162,7 @@ type options struct {
 	tlshshaketimeout time.Duration
 	client           HTTPClient
 	httpHeaders      map[string]string
+	MTOM             bool
```

Comment:
Let's make this field private as the others.

Reply:
Done